src/flamegraph/color.rs

```diff
@@ -17,14 +17,15 @@ const GRAY_GRADIENT: (&str, &str) = ("#f8f8f8", "#e8e8e8");
 
 #[derive(Debug, PartialEq)]
 pub enum Palette {
+    Basic(BasicPalette),
+    Multi(MultiPalette),
+}
+
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
I would also `derive(Copy)` on this.

Reply:
[Ok](https://github.com/jonhoo/inferno/pull/32/commits/40386d59b8986d0089d0c998c339915be353062a)